_posts/2024-08-01-contributing-to-che-docs.adoc

```diff
@@ -0,0 +1,51 @@
+---
+title: Contributing to Eclipse Che documentation
+layout: post
+author: Jana Vrbkova
+description: >-
+  Contribute to Eclipse Che documentation using an Eclipse Che workspace.
+categories: []
+keywords: ['che-docs', 'documentation', 'write', 'contribute']
+slug: /@deerskindoll/contributing-to-che-docs
+---
+
+= Contributing to Eclipse Che documentation
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
```

Comment:
Please add `Devfile and How it Works?` section or smth. similar that would explain CDE, predefined commands etc.

Reply:
image that might be useful here 
<img width="1137" alt="Screenshot 2024-07-26 at 12 27 07" src="https://github.com/user-attachments/assets/4b561bb4-d22d-4a28-bc15-c5833b540506">